templates/android.yml

```diff
@@ -7,7 +7,7 @@ trigger:
 - master
 
 pool:
-  vmImage: 'macOS-10.13'
+  vmImage: 'macOS-latest'
```

Comment:
Same for Mac and Linux... wish they were all lowercase, but it's a nit

Reply:
+1 for all lower case.